DNN Platform/Library/Entities/Users/UserController.cs

```diff
@@ -1045,8 +1045,9 @@ public static bool DeleteUser(ref UserInfo user, bool notify, bool deleteAdmin)
                     //send email notification to portal administrator that the user was removed from the portal
                     SendDeleteEmailNotifications(user, portalSettings);
                 }
-                DataCache.ClearPortalCache(portalId, false);
-                DataCache.ClearUserCache(portalId, user.Username);
+
+                DataCache.ClearPortalCache(user.PortalID, false);

```

Comment:
As we are making changes in this section, is there a reason that we clear the entire portal cache when we delete a user?  I thought we had a recent change to avoid this. @dnnsoftware/tag

Reply:
Indeed. You are referring to #2175. We could apply an analogous change here

Reply:
Thnx @tpluscode. Let me check this, if it works then would push those changes